src/models/RatesByAssetSymbol.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2021 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+// constants
+import { ETH } from 'constants/assetsConstants';
+
+// types
+import type { ChainRecord } from 'models/Chain';
+import type { Currency } from 'models/Asset';
+
+
+export type RateByCurrencySymbol = { [currencySymbol: string]: number };
```

Comment:
Consider changing key name, as `currencySymbol` is not exact, because we have USD, EUR, GBP & ETH, so last one is not exactly currency symbol.

```suggestion
// TODO: refactor rateKey to be of RateKey type.
export type Rates = { [rateKey: string]: number };
```

Reply:
Consider moving `export type RateKey = Currency | typeof ETH;` next to this type because they are linked.